scripts/start.sh

```diff
@@ -64,13 +64,17 @@ fi
 # Argument passed to dev_appserver.py to indicate whether or not to
 # clear the datastore.
 CLEAR_DATASTORE_ARG="--clear_datastore=true"
+ENABLE_CONSOLE_ARG=""
 
 # Argument passed to feconf.py to help choose production templates folder.
 FORCE_PROD_MODE=False
 for arg in "$@"; do
   if [ "$arg" == "--save_datastore" ]; then
     CLEAR_DATASTORE_ARG=""
   fi
+  if [ "$arg" == "--enable_console" ]; then
+    ENABLE_CONSOLE_ARG="--enable_console=true"
```

Comment:
What is this for & how does it relate to the skills work? Can this be done as a separate PR if it's unrelated?

Reply:
This is not related to this PR in particular. For enabling developer console at localhost:8000 (which was what I used to test Questions as the frontend is not present), this extra argument was required, so I made this change, so that in the future anyone can enable developer console by just adding this argument when running start.sh.

Reply:
Ah. Can you put it in a separate PR then? That way, you can include the rationale for the change in the PR description and it'll be easier to understand when looking back on it in future.

Reply:
Ok, will do that.

Reply:
Done, removed it, will create a separate PR for this alone.